docs/reference/setup/install/docker.asciidoc

```diff
@@ -0,0 +1,290 @@
+[[docker]]
+=== Install Elasticsearch with Docker
+
+Elasticsearch is also available as a Docker image.
+The image is built with https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack].
+
+==== Security note
+
+NOTE: https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack] is preinstalled in this image.
+Please take a few minutes to familiarize yourself with https://www.elastic.co/guide/en/x-pack/current/security-getting-started.html[X-Pack Security] and how to change default passwords. The default password for the `elastic` user is `changeme`.
+
+NOTE: X-Pack includes a trial license for 30 days. After that, you can obtain one of the https://www.elastic.co/subscriptions[available subscriptions] or https://www.elastic.co/guide/en/x-pack/current/security-settings.html[disable Security]. The Basic license is free and includes the https://www.elastic.co/products/x-pack/monitoring[Monitoring] extension.
```

Comment:
Fixed in https://github.com/elastic/elasticsearch/pull/21497/commits/d7646bdbb166a0781130f558637da93214c3ec23